core/src/components/segment/segment.md.scss

```diff
@@ -40,6 +39,11 @@
   color: #{current-color(base)};
 }
 
+@media (any-hover: hover) {
+  :host(.ion-color)::slotted(:hover:not(.segment-button-checked)) {
```

Comment:
This doesn't seem to be working for selected segments w/ color.

Reply:
Yeah that has been intentional, I'm sure if hover should work on one that is already checked.

Reply:
Hmm interesting. It looks like the hover is working for selected segments that just default to the primary color.

Reply:
Oh sorry, I thought this was iOS. I see it on MD.